pkg/reconciler/route/controller.go

```diff
@@ -86,9 +88,8 @@ func NewControllerWithClock(
 		Handler:    controller.HandleAll(impl.EnqueueControllerOf),
 	})
 
-	clusterIngressInformer.Informer().AddEventHandler(controller.HandleAll(
-		impl.EnqueueLabelOfNamespaceScopedResource(
-			serving.RouteNamespaceLabelKey, serving.RouteLabelKey)))
+	clusterIngressInformer.Informer().AddEventHandler(controller.HandleAll(impl.EnqueueControllerOf))
```

Comment:
This cannot be `EnqueueControllerOf` because it is cluster scoped.

Reply:
Fixed.